packages/ExternalLink/src/ExternalLink.styles.js

```diff
@@ -3,21 +3,23 @@ import tokens from "@paprika/tokens";
 import { truncateText } from "@paprika/stylers/lib/includes";
 import { toInt } from "@paprika/stylers/lib/helpers";
 
-export const ExternalLink = styled.a`
+export const ExternalLinkWrapper = styled.a`
+  ${props => `
   align-items: center;
   border-radius: ${tokens.border.radius};
   color: ${tokens.textColor.link};
   display: inline-flex;
   max-width: 100%;
   position: relative;
 
-  ${({ iconFontSize }) => `padding: 1px ${iconFontSize + toInt(tokens.spaceLg)}px 1px ${tokens.spaceSm};`}
+  ${`padding: 1px ${props.iconFontSize + toInt(tokens.spaceLg)}px 1px ${tokens.spaceSm};`}
```

Comment:
I think when the prop is _only_ being used in this one spot it makes more sense _not_ to wrap the whole styling block in a function, but keep it scoped to the one line, as it was.  

Gets to be a bit of a judgement call then about when there's only _two_ lines... or _three_ lines... when to wrap the whole block... what do you think?

Reply:
Does line 7 also need `css` for syntax highlighting?

I'm ambivalent but just sharing some links. Overall this approach seems similar to this, correct? https://twitter.com/NikkitaFTW/status/1262423045874089990
And FWIW I found another one in the comments: https://twitter.com/Saeris/status/1262441528078688256

Reply:
I am interested in your ambivalent feelings... or do you really mean _indifferent_?

I actually really like the suggestion from @saeris... so this would look more like:

```jsx
export const ExternalLink = styled.a(props) => css`
  ...
`;
```

or 

```jsx
export const ExternalLink = styled.a({ someProp, someOtherProp }) => css`
  ...
`;
```
```

Also, yes, line 7 _should_ include the `css` helper... I didn't comment on every instance throughout the PR, it comes up for nearly every component.